src/server/xtra2.c

```diff
@@ -1446,36 +1509,43 @@ bool set_cut(int Ind, int v)
 			/* Graze */
 			case 1:
 			msg_print(Ind, "You have been given a graze.");
+			sound(Ind, MSG_CUT);
```

Comment:
I'm assuming the cut noise doesn't get louder based on how bad it is. How about we call it once?